### DISCUSSION TITLE
Calcite can support kvrocks

### DISCUSSION THREAD

**Discussion by caicancai** [apache/kvrocks#2532] [General]
I noticed that kvrocks already supports SQL syntax, and calcite currently supports redis. If the SQL protocol of kvrocks and redis is the same, then calcite will be able to easily support kvrocks. I can try to promote this work in calcite.

I am not very familiar with kvrocks at present, and I would like to ask for your suggestions

**Comment by caicancai** [comment]
cc @PragmaTwice @git-hulk

**Reply by git-hulk** [reply]
It's a good proposal, very happy to see this happen.

**Comment by caicancai** [comment]
@git-hulk I will start working on this part, thanks
